src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs

```diff
@@ -12747,6 +12747,83 @@ class C
             Directory.Delete(dir.Path, true);
         }
 
+        [Fact]
+        public void SourceGenerators_GeneratedDir_Has_Spaces()
```

Comment:
This test would actually pass without the fix, but I figured it was worth adding the extra coverage anyway.